apps/search/api/serializers.py

```diff
@@ -12,3 +16,15 @@ class SearchableModelSerializer(ModelSerializer):
     tagsHtml = serpy.StrField(attr='tags_html')
     title = serpy.StrField()
     verified = serpy.BoolField()
+    meta = serpy.MethodField()
+
+    def get_meta(self, model) -> Optional[dict]:
+        if not hasattr(model, 'meta'):
+            return None
+
+        meta = model.meta.to_dict()
+        for key in ELASTICSEARCH_META_FIELDS_TO_CLEAN:
+            # TODO: find out why in some cases it's already cleaned
```

Comment:
If you want, we could use a try-except:
```
try:
    del meta[key]
except KeyError as err:
    logging.info(
        f'The "{key}" key has already been removed from model meta.'
    )
```

Reply:
I would like to keep logs clean for now because these always occur.